rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -14,11 +14,21 @@ const Landing = () => {
     if (!isValidNestedPath) history.push(routePaths.landing);
   }
 
+  const listOfRigs = [
+    {"id": "asbhkjsahb278173",
```

Comment:
don't know if this is you IDE or prettier making this change but can we fix this object structure?